addons/website/models/ir_http.py

```diff
@@ -133,7 +133,7 @@ def _auth_method_public(cls):
 
     @classmethod
     def _register_website_track(cls, response):
-        if getattr(response, 'status_code', 0) != 200:
+        if getattr(response, 'status_code', 0) != 200 or request.httprequest.headers.get('X-Do-Not-Track'):
```

Comment:
Although it's mostly been dropped as a concept, maybe the [DNT header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/DNT) could be used instead? It has the advantage of well-defined semantics, and respecting it for the 3 users which do use it is nice.